### DISCUSSION TITLE
can't export inferno and italy

### DISCUSSION THREAD

**Discussion by takayamamaria** [SteamDatabase/ValveResourceFormat#614] [General]
![image](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/116933675/1a202d21-abbe-4c0a-bbb9-cf32af24c04a)
![image](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/116933675/84a1f86e-6bee-4c35-87dd-22ef3ec52f57)

hi I was try to export the de_italy and de_inferno.but have error,
how to do it,just wait for new versions?

**Comment by xPaw** [comment]
Try latest unstable.
